lib/runners/cli.rb

```diff
@@ -63,14 +63,14 @@ def run
 
         result.tap do
           finished_at = Time.now
-          trace_writer.header "Analysis finished", recorded_at: finished_at
+          trace_writer.header "Analysis finished"
 
           if result.is_a? Results::Success
             trace_writer.message "#{result.issues.size} issue(s) found."
           end
 
           trace_writer.message "Finished at #{finished_at.utc}"
-          trace_writer.message "Elapsed time: #{format_duration(finished_at - started_at)}"
+          trace_writer.finish "Elapsed time: #{format_duration(finished_at - started_at)}", duration_in_sec: Integer(finished_at - started_at)
```

Comment:
[suggestion] How about keeping the existing `#message` and adding `#finish` without a message?
If doing so, Runners clients don't have to change the code to show the finish message.

```suggestion
          trace_writer.message "Elapsed time: #{format_duration(finished_at - started_at)}"
          trace_writer.finish started_at: started_at, finished_at: finished_at, duration_in_sec: Integer(finished_at - started_at)
```

Reply:
When it comes to Sider, `finish` without `message` seems to require us to change our code because we just see the key, `"trace"`, to extract traces from the output. Perhaps, we should exclude traces without `message`. Is it OK?

Reply:
Sorry, I can't understand the following: 😓 

> Perhaps, we should exclude traces without `message`.

What code do you mean?

Reply:
16d328ba